server/functions.lua

```diff
@@ -223,31 +260,39 @@ end
 QBCore.Functions.IsPlayerBanned = function (source)
 	local retval = false
 	local message = ""
-    local result = exports.ghmattimysql:executeSync('SELECT * FROM bans WHERE license=@license', {['@license'] = QBCore.Functions.GetIdentifier(source, 'license')})
-    if result[1] ~= nil then
-        if os.time() < result[1].expire then
-            retval = true
-            local timeTable = os.date("*t", tonumber(result.expire))
-            message = "You have been banned from the server:\n"..result[1].reason.."\nYour ban expires "..timeTable.day.. "/" .. timeTable.month .. "/" .. timeTable.year .. " " .. timeTable.hour.. ":" .. timeTable.min .. "\n"
-        else
-            exports['ghmattimysql']:execute('DELETE FROM bans WHERE id=@id', {['@id'] = result[1].id})
-        end
-    end
+	local result = exports.ghmattimysql:executeSync('SELECT * FROM bans WHERE license = @license', {['@license'] = QBCore.Functions.GetIdentifier(source, 'license')})
+
+	if result[1] ~= nil then
+		if os.time() < result[1].expire then
+			retval = true
+
+			local timeTable = os.date("*t", tonumber(result.expire))
+
+			message = "You have been banned from the server:\n" .. result[1].reason .. "\nYour ban expires " .. timeTable.day .. "/" .. timeTable.month .. "/" .. timeTable.year .. " " .. timeTable.hour .. ":" .. timeTable.min .. "\n"
```

Comment:
For better order: `message = "You have been banned from the server:\n" .. result[1].reason .. "\nYour ban expires " .. timeTable.month .. "/" .. timeTable.day .. "/" .. timeTable.year .. " " .. timeTable.hour .. ":" .. timeTable.min .. "\n"`

Using date and time notation in the United States correctly.